src/app/features/test-records/create/views/create-test-record/create-test-record.component.ts

```diff
@@ -103,17 +113,26 @@ export class CreateTestRecordComponent implements OnInit, OnDestroy, AfterViewIn
     this.testRecordsService.createTestResult(cloneDeep(testResult));
   }
 
-  handleReview() {
-    if (this.isAnyFormInvalid()) {
-      return;
-    }
+  async handleReview() {
+    if (this.isAnyFormInvalid()) return;
+    if (this.techRecord == null) return;
 
     this.errorService.clearErrors();
     this.testMode = TestModeEnum.View;
+
+    const testResult = await firstValueFrom(this.testResult$);
+    if (testResult && this.techRecord?.techRecord_statusCode === StatusCodes.PROVISIONAL
+      && this.testTypeId
+      && this.validateUpdateStatus(testResult.testTypes[0].testResult, this.testTypeId)) {
+      const warnings: GlobalWarning[] = [];
```

Comment:
I just think if we are going this route of all this engineering to tell them to refresh, we may as well refresh for them

Reply:
Or even lets do a call once to see if we have a new current, if we don't warning, if we do, just get it

Reply:
Good point, I did try checking state initially, but the network request which fetches the test result history happens in a resolver or something, so it isn't available in state immediately after the test result is created - this way just seemed like a way to guarantee we had the information. I could probably add both, with the delay, to avoid making the request if not needed?

I think the refresh thing still wouldn't fully work as this would make the results history appear correct, but the url wouldn't change so you'd still be viewing the now archived provisional record?

Reply:
Yeah sorry refresh was a bad turn of phrase. Honestly I just tested it and it's fine like this. Going to approve.